src/api/spec/features/webui/requests_spec.rb

```diff
@@ -231,14 +231,24 @@ def branch_and_change_sources
   context 'review' do
     describe 'for user' do
       let(:reviewer) { create(:confirmed_user) }
-      it 'opens a review' do
+
+      it 'opens a review and accepts it' do
         login submitter
         visit request_show_path(bs_request)
         click_link 'Add a review'
         find(:id, 'review_type').select('User')
         fill_in 'review_user', with: reviewer.login
         click_button 'Ok'
         expect(page).to have_text("Open review for #{reviewer.login}")
+        expect(page).to have_text("Request 1 (review)")
```

Comment:
I could add something along these lines to test its been created:
```ruby
expect(Review.all.count).to eq(1)
```

However IMO line 243 is already checking that a review with an id=1 exists doesn't it?